packages/web-frontend/src/containers/UserMenu/index.js

```diff
@@ -45,10 +45,6 @@ class UserMenu extends Component {
     };
   }
 
-  componentDidMount() {
-    this.props.onRefreshCurrentUser();
-  }
```

Comment:
I couldn't find a reason that the user should only log in if the UserMenu renders, can you?

Reply:
I can't find either!

Reply:
This is the commit that added the above (3 years ago!): https://github.com/beyondessential/tupaia-web/commit/6fd61f54077e600dafea12ed4f35954e6b933ce3#diff-5408a22f0a6eb54adba94ad67d94d505

Reply:
Wow. Good find! Simpler times...